projects/bagel-finance/index.js

```diff
@@ -62,11 +74,11 @@ async function tvl(timestamp, ethBlock, chainBlocks) {
   }).map((n) => {
     const stakingTokenAddr = n.input.params[0];
     const i5 = n.output['5'];
-    //const i6 = n.output['6'];
+    const i6 = n.output['6'];  
     const i8 = n.output['8'];
     return {
       token: stakingTokenAddr,
-      balance: BigNumber(i5).plus(BigNumber(i8)).toFixed(0)
+      balance: BigNumber(i5).plus(BigNumber(i6)).plus(BigNumber(i6)).plus(BigNumber(i8)).toFixed(0)
```

Comment:
doesn't i6 count borrowed money, which is used on the lps (which we are also counting)? therefore i'm worried that this may lead to double counting (count money borrowed from the pools once, and then again when counting the money in lps)

Reply:
Hi sir, the TVL calculated by : Leding + Borrwoing + LP.
i6 is for Borrowing, maybe it will be double counted by LP, cuz when the assets borrowed, user need to depostied the same value equivalent to borrowing into our platform. So it might be counted by LP too. But we can't change this contract , so could you pls update it in accordance with this calculation?
So that's the reason why TVL shows 15M+ in our website but shows 8M in Defi llama.

Reply:
If i6 is for borrowing, doesn't that money get borrowed into the LPs?
the issue that i'm worried about is that:
- I have 1 USD and deposit it into a pool
- Then someone else borrows and LPs with it
- Then I'll count the value of the LP staked (including 1$) and on top of that the value of the 1$ borrowed
- the 1$ i provided gets counted twice
is this correct?